src/external_secondaries/ecuinterface.h

```diff
@@ -0,0 +1,18 @@
+#include <string>
+
+class ECUInterface {
+ public:
+  enum InstallStatus {
+    InstallationSuccessfull = 0,
```

Comment:
Minor typo: This should be `InstallationSuccessful` (yes, English spelling is ridiculous)